.github/workflows/ci.yml

```diff
@@ -94,85 +94,7 @@ jobs:
-    services:
-      mysql:
-        image: mysql:5.7
-        env:
-          MYSQL_ALLOW_EMPTY_PASSWORD: false
-          MYSQL_ROOT_PASSWORD: testpassword
-          MYSQL_DATABASE: test_db
-        ports:
-          - 3306/tcp
-        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Setup PHP Extension Cache
-        id: cache-env
-        uses: shivammathur/cache-extensions@v1
-        with:
-          php-version: ${{ env.php_version }}
-          extensions: ${{ env.php_extensions }}
-          key: php80-ext-cache-${{ hashFiles('.github/workflows/ci.yml') }}
-
-      - name: Cache PHP Extensions
-        uses: actions/cache@v4
-        with:
-          path: ${{ steps.cache-env.outputs.dir }}
-          key: ${{ steps.cache-env.outputs.key }}
-          restore-keys: ${{ steps.cache-env.outputs.key }}
-
-      - name: Setup PHP, with composer and extensions
-        uses: shivammathur/setup-php@v2
-        with:
-          php-version: ${{ env.php_version }}
-          extensions: ${{ env.php_extensions }}
-          ini-values: log_errors=On, error_log="${{github.workspace}}/artifacts/php_errors.log"
-          coverage: none
-          tools: composer:v2
-
-      - name: Get composer cache directory
-        id: composer-cache
-        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
-
-      - name: Cache composer dependencies
-        uses: actions/cache@v4
-        with:
-          path: ${{ steps.composer-cache.outputs.dir }}
-          key: ${{ runner.os }}-composer-ss413-php80-${{ hashFiles('**/composer.json') }}-${{ hashFiles('.github/workflows/ci.yml') }}
-          restore-keys: ${{ runner.os }}-composer-ss413-php80-
-
-      - name: Install Composer dependencies
-        env:
-            GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          composer self-update --ansi || true
-          if [[ $GITHUB_ACCESS_TOKEN ]]; then composer config --ansi -g github-oauth.github.com $GITHUB_ACCESS_TOKEN; fi
-          composer config allow-plugins.composer/installers true
-          composer config allow-plugins.silverstripe/recipe-plugin true
-          composer config allow-plugins.silverstripe/vendor-plugin true
-          composer require silverstripe/recipe-cms 4.13.*
-          composer install --prefer-dist --verbose --no-interaction --no-progress --no-suggest --optimize-autoloader --ansi
-
-      - name: Configure Environment
-        run: |
-          cp tests/utils/actions.env.template .env
-          mkdir artifacts
-
-      - name: Perform PHPUnit Tests
-        env:
-          SS_DATABASE_PORT: ${{ job.services.mysql.ports['3306'] }}
-        run: vendor/bin/phpunit --colors=always --teamcity
+        run: SS_PHPUNIT_FLUSH=1 vendor/bin/phpunit --colors=always --teamcity
```

Comment:
This doesn't work in SS4 you'll need to use the old method in the phpunit.xml.dist file